demisto_sdk/commands/generate_docs/generate_playbook_doc.py

```diff
@@ -86,14 +86,42 @@ def get_playbook_dependencies(playbook):
     playbooks = set()
 
     playbook_tasks = playbook.get('tasks')
+    playbook_path = os.path.relpath(input)
+    pack_path = os.path.dirname(os.path.dirname(playbook_path))
+    integration_dir_path = os.path.join(pack_path, 'Integrations')
+    # Get all files in integrations directories
+    pack_files = [os.path.join(r, file) for r, d, f in os.walk(integration_dir_path) for file in f]
+    integrations_files = []
+    for file in pack_files:
+        if file.endswith('.yml') and not os.path.basename(file).startswith('integration'):
```

Comment:
can you explain the second part of the condition?

Reply:
yes. This is for unify files. If we have unify in the package it will start with the prefix, wanted to ignore that.